### DISCUSSION TITLE
Chakra is overriding global H1...H6, makes it impossible for third party components to work

### DISCUSSION THREAD

**Discussion by androidplayer1** [chakra-ui/chakra-ui#8287] [General]
### Description

![image](https://github.com/chakra-ui/chakra-ui/assets/86848862/c2c1e928-d0fb-4196-8550-5323828ec91e)

### Link to Reproduction

https://google.com

### Steps to reproduce

I am trying to use a markdown editor, but that is not possible due to Chakra intefering/overriding the H1, H2, H3, H4, H5, H6 styling to every element globally.

### Chakra UI Version

Latest

### Operating System

- [X] macOS
- [X] Windows
- [ ] Linux

**Comment by jamesarosen** [comment]
https://github.com/chakra-ui/chakra-ui/issues/4427 may help

**Comment by segunadebayo** [comment]
Preflight resets `h1`–`h6`. Scope the reset, or stop styling raw headings in `globalCss`.
